spec/like_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject do
+    user = User.new(name: 'Javier', photo: 'https://whereever.com', bio: 'Surfer', posts_counter: 0)
+    Post.create(title: 'Good Habits', text: 'This is why habits are necessary.', comments_counter: 2,
+                likes_counter: 5, user:)
+  end
```

Comment:
- [ ] [REQUIRED] Let's fix that same syntax error I mentioned earlier here as well.
You need to pass a valid value for the user parameter. This should be a reference to a User instance that you have created in your test.

Kindly refer to the recommendation on made in the `comment_spec.rb`